tests/README.md

```diff
@@ -0,0 +1,51 @@
+## Testing in PTB
+
+PTB uses [pytest](https://docs.pytest.org/en/latest/) for testing. To run the tests, you need to
+have pytest installed along with a few other dependencies. You can find the list of dependencies
+in the `requirements-dev.txt` file in the root of the repository.
+
+### Running tests
+
+To run the entire test suite, you can use the following command:
+
+    $ pytest
+
+This will run all the tests, including the ones which make a request to the Telegram servers, which
+may take a long time (about > 13 mins). To run only the tests that don't require a connection, you
+can run the following command:
+
+    $ pytest -m no_req
+
+To further speed up the tests, you can run them in parallel using the `-n` flag:
```

Comment:
please explain that this uses multiple CPU cores. May be a no-go for people on potato machines